releasetarball/compile_test.go

```diff
@@ -182,14 +182,29 @@ var _ = Describe("Compile", func() {
 	})
 
 	When("compiling a release with multiple source packages", func() {
+		var multiplePackageCompiler *fakes.Compiler
 		BeforeEach(func() {
 			sourceTarballPath = filepath.Join("testdata", "log-cache-release-3.0.9.tgz")
+
+			multiplePackageCompiler = new(fakes.Compiler)
+			Expect(os.WriteFile(filepath.Join(d.BlobsDir(), "golang-1.20-linux-blob"), []byte("golang-1.20-linux-blob"), 0644)).NotTo(HaveOccurred())
```

Comment:
It may be clearer to write this as `Expect(s.WriteFile(...)).To(Succeed())`.

Reply:
Cool! That is clearer. I forgot that matcher was a thing. I generally avoid Gomega/Gingo on my own projects so I forget how expressive it can be.

Reply:
I pushed the refactor. I found a few other assertions that could use the `To(Succeed())` pattern and I converted the block you mentioned into a loop.

Reply:
Cool, thanks!